src/amo/css/AddonReviewList.scss

```diff
@@ -35,6 +37,7 @@
       text-decoration: none;
       color: $primary-font-color;
       font-weight: normal;
+      word-wrap: break-word;
```

Comment:
Could you change this to `overflow-wrap`? I feel like we might be using `word-wrap` elsewhere but we shouldn't 😅 .

This is a bit of a nitpick as [`word-wrap` aliases to `overflow-wrap`](https://developer.mozilla.org/en-US/docs/Web/CSS/overflow-wrap) but it would be nice to use the "proper" attribute. 😄